safe/gui/tools/batch/batch_dialog.py

```diff
@@ -679,7 +741,7 @@ def on_output_directory_chooser_clicked(self):
     def help_toggled(self, flag):
         """Show or hide the help tab in the main stacked widget.
 
-        .. versionadded: 3.2.1
+        .. version added: 3.2.1
```

Comment:
For next time, it was correct without a space. See http://www.sphinx-doc.org/en/1.4.8/markup/para.html#directive-versionadded